app/src/lib/git/rebase.ts

```diff
@@ -2,7 +2,7 @@ import * as Path from 'path'
 import { ChildProcess } from 'child_process'
 import * as FSE from 'fs-extra'
 import { GitError } from 'dugite'
-import * as byline from 'byline'
+import byline from 'byline'
```

Comment:
I've had to change all the `import *` statements that were invalid based on the ESM standard (which mandates that the output of import star is an object with the exported values of the object).

Good thing is that with the interop flag, TypeScript now complains about this 😃